tests/snmpsim/technicolor_tg650s.snmprec

```diff
@@ -0,0 +1,3 @@
+1.3.6.1.2.1.1.1.0|4|TG650s
```

Comment:
Is this the full sysDescr and sysObjectID from the devices?

Reply:
sysDescr is OK
sysObjectID is 1.3.6.1.4.1.2863.405.1

Fixed by new commit